### PR TITLE
Necropolis and mine walls

### PR DESCRIPTION
**Pull request by NULL882**
## About the PR

For https://github.com/space-wizards/space-station-14/issues/14226

Just walls for some lavaland structures.

## Why / Balance

lavaland wall assets.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image
image

## Breaking changes

**Changelog**

:cl: NULL882
- add: Added Necropolis Wall.
- add: Added Mine Wall.
- add: Added Sings for Mine Wall.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml [hunk 1]
```diff
@@ -1273,3 +1273,21 @@
   components:
   - type: Sprite
     state: zero
+
+- type: entity
+  parent: BaseSign
+  id: SignSurvival
+  name: survival sign
+  description: A sign, survivals is written on it.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml:1281]:
```suggestion
  description: A sign. "Survival" is written on it.
```

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml [hunk 2]
```diff
@@ -1273,3 +1273,21 @@
...
+  components:
+  - type: Sprite
+    state: survival
+
+- type: entity
+  parent: BaseSign
+  id: SignNTMine
+  name: mine sign
+  description: A sign, mine is written on it.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml:1290]:
```suggestion
  description: A sign. "Mine" is written on it.
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]
all the actual stuff looks good

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml (hunk 2 above)
Comment (NULL882) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml]: Done.

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml (hunk 1 above)
Comment (NULL882) [reply Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml]: Done.

**Review by mirrorcult — APPROVED** [review]

**Comment by Carolyn3114** [comment]
wait, how's there a watcher there?

**Comment by mirrorcult** [comment]
we have watchers

**Comment by Carolyn3114** [comment]
must be new

**Pull request merged**